ext/oci8/oci8_interface.c

```diff
@@ -640,15 +635,15 @@ PHP_FUNCTION(oci_lob_erase)
 	if (offset_is_null) {
 		offset = -1;
 	} else if (offset < 0) {
-		php_error_docref(NULL, E_WARNING, "Offset must be greater than or equal to 0");
-		RETURN_FALSE;
+		zend_argument_value_error(2, "must be greater than or equal to 0");
```

Comment:
The '2' is off-by-one for [`$lob->erase()`](https://www.php.net/manual/en/oci-lob.erase.php) but is valid for the `oci_lob_erase()`.  For `$lob->erase(-10)` the resulting message from the PR refers to `$length` where it should be `$offset`, which is the first arg.

Since `oci_lob_erase()` is not documented, the simple solution is to subtract one from the parameter number and ignore the wrong message if anyone is actually using `oci_lob_erase()` incorrectly .  Or is there a way to distinguish between the method and function usage?

The same problem occurs for the next `zend_argument_value_error()` call, and others in this file.